checkov/arm/graph_builder/local_graph.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import annotations
+
+import logging
+from copy import deepcopy
+from typing import Any, TYPE_CHECKING
+
+from checkov.arm.graph_builder.graph_components.block_types import BlockType
+from checkov.arm.graph_builder.graph_components.blocks import ArmBlock
+from checkov.arm.utils import ArmElements
+from checkov.common.graph.graph_builder import CustomAttributes
+from checkov.common.graph.graph_builder.graph_components.blocks import Block
+from checkov.common.graph.graph_builder.local_graph import LocalGraph
+from checkov.common.util.consts import START_LINE, END_LINE
+
+if TYPE_CHECKING:
+    from checkov.common.graph.graph_builder.local_graph import _Block
+
+
+class ArmLocalGraph(LocalGraph[Block]):
+    def __init__(self, definitions: dict[str, dict[str, Any]]) -> None:
+        super().__init__()
+        self.vertices: list[Block] = []
```

Comment:
doesn't it included in the `super().__init__()`?

Reply:
I need to call it otherwise the attributes in the parent won't be initialized.